adyenv6core/src/com/adyen/v6/forms/AdyenPaymentForm.java

```diff
@@ -320,6 +340,8 @@ public String toString() {
         sb.append("    cardType: ").append(Util.toIndentedString(cardType)).append("\n");
         sb.append("    installments: ").append(Util.toIndentedString(installments)).append("\n");
         sb.append("    issuerId: ").append(Util.toIndentedString(issuerId)).append("\n");
+        sb.append("    sepaOwnerName: ").append(Util.toIndentedString(sepaOwnerName)).append("\n");
```

Comment:
do we need to reset the two values in the `resetFormExceptBillingAddress` function above?

Reply:
yes, added :)